src/app/(proper_react)/redesign/(public)/PlansTable.module.scss

```diff
@@ -0,0 +1,242 @@
+@import "../../../tokens";
+
+.plansTable {
+  border-spacing: $spacing-xl 0;
+  table-layout: fixed;
+  width: 100%;
+
+  thead th {
+    width: calc(30% - $spacing-xl);
+  }
+
+  td {
+    padding-block: $spacing-md;
+
+    .cellWrapper {
+      display: flex;
+      align-items: center;
+    }
+
+    svg.checkIcon {
+      display: inline-block;
+      color: $color-blue-50;
+    }
+  }
+
+  thead th:nth-child(1) {
+    text-align: start;
+    vertical-align: bottom;
+    padding-block: $spacing-lg;
+  }
+
+  thead th:nth-child(2),
+  thead th:nth-child(3),
+  tbody td:nth-child(2),
+  tbody td:nth-child(3) {
+    background-color: $color-white;
+    vertical-align: middle;
+    color: $color-grey-50;
+
+    .cellWrapper {
+      justify-content: center;
+    }
+  }
+
+  thead th:nth-child(3),
+  tbody td:nth-child(3) {
+    font-weight: 700;
+    border-inline: 4px solid $color-purple-70;
+  }
+
+  thead th:nth-child(3) {
+    border-top: 4px solid $color-purple-70;
+    border-top-right-radius: $border-radius-md;
+    border-top-left-radius: $border-radius-md;
+
+    &::before {
+      content: "+";
+      position: absolute;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: $color-purple-70;
+      color: $color-white;
+      width: $spacing-2xl;
+      height: $spacing-2xl;
+      border-radius: $border-radius-xl;
+      font: $text-title-sm;
+      inset-block-start: 0;
+      inset-inline-start: 0;
+      transform: translateX(-50%) translateY(-50%);
+    }
+  }
+
+  tbody tr:last-child td:nth-child(3) {
+    border-bottom: 4px solid $color-purple-70;
+    border-bottom-right-radius: $border-radius-md;
+    border-bottom-left-radius: $border-radius-md;
+  }
+
+  // This rule is intentionally after `tbody tr:last-child td:nth-child(3)`,
+  // which puts the purple border around the "Plus" column. It's bound to be
+  // specific enough to override later styles, and makes sense to be close to
+  // other highlight-related code:
+  /* stylelint-disable-next-line no-descending-specificity */
+  thead th:nth-child(2),
+  tbody td:nth-child(2) {
+    border-inline: 1px solid $color-grey-20;
+  }
+
+  thead th:nth-child(2) {
+    border-top: 1px solid $color-grey-20;
+    border-top-right-radius: $border-radius-md;
+    border-top-left-radius: $border-radius-md;
+  }
+
+  tbody tr:last-child td:nth-child(2) {
+    border-bottom: 1px solid $color-grey-20;
+    border-bottom-right-radius: $border-radius-md;
+    border-bottom-left-radius: $border-radius-md;
+  }
+
+  thead {
+    th:nth-child(2),
+    th:nth-child(3) {
+      padding-block: $spacing-2xl;
+      position: relative;
+    }
+
+    .badge {
+      position: absolute;
+      top: 0;
+      background-color: $color-purple-70;
+      color: $color-white;
+      text-transform: uppercase;
```

Comment:
@flodolo Did we have issues elsewhere about transforming text casing via CSS vs directly in the .ftl files.

Reply:
Yep https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues

Reply:
Hmm, but I get the exact opposite advice for accessibility: let CSS handle stylistic capitalisation, and use normal text for the source. I believe to avoid screen readers trying to read it out as an acronym. Maybe I can make this rule only apply for English, and just sentence-casing for other locales?

Reply:
> Maybe I can make this rule only apply for English, and just sentence-casing for other locales?

That would work for me.

Reply:
Cool, applied in 8dece1f36.